viewer/src/main/webapp/viewer-html/components/Edit.js

```diff
@@ -1074,10 +1079,26 @@ Ext.define("viewer.components.Edit", {
             input.addCls("x-item-disabled");
         }
 
+        input.on({
+            change: this.closureOnFormFieldChange(this)
+        });
+
         return input;
     },
-    setInputPanel: function (feature) {
+    closureOnFormFieldChange: function (me) {
+        return function (input, newValue, oldValue) {
```

Comment:
See comment when adding listener, this can then be rewritten as:
```
closureOnFormFieldChange: function (input, newValue, oldValue) {
    if (!this.formValuesAreBeingUpdated && this.mode === 'edit') {
        this.onFormFieldChange(input, newValue, oldValue)
    }
},
```
Then its probably better to rename the function to something like `validateFormChange`